tests/chainer_tests/links_tests/loss_tests/test_negative_sampling.py

```diff
@@ -88,88 +103,28 @@ def test_to_cpu(self):
         self.link.to_cpu()
         self.assertFalse(self.link.sampler.use_gpu)
 
-    @attr.gpu
-    def test_backward_cpu_gpu(self):
```

Comment:
The original test is check if result of cpu is same as that of gpu. Instead i checked results of both cpu and gpu directly. I think the original one is redundant.